texi2rst.py

```diff
@@ -1658,6 +1740,20 @@ def test_subsubheading(self):
         self.assertEqual(u'A sub-sub-heading\n^^^^^^^^^^^^^^^^^\n\nsome text\n\n',
                          out)
 
+    def test_option_ref_with_var(self):
+        xml_src = '<para>This is similar to how <option>-Walloca-larger-than=</option><var>byte-size</var> works.</para>'
+        doc = from_xml_string(xml_src)
+        doc = convert_to_rst(doc, self.ctxt)
+        out = self.make_rst_string(doc)
+        self.assertEqual('This is similar to how :option:`-Walloca-larger-than`:samp:`=byte-size` works.\n\n', out)
```

Comment:
Can this be:

    :option:`-Walloca-larger-than`:samp:`={byte-size}`

The ``:samp:`` role has special handling for stuff in braces that convey the variable part:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-samp

Reply:
Implemented that.